app/controllers/shelters_controller.rb

```diff
@@ -26,6 +26,7 @@ def create
 
       if draft.save
         redirect_to draft, notice: 'Your new shelter is pending approval.'
+        send_slack_notification "New shelter added: https://irma-api.herokuapp.com/drafts/#{draft.id}"
```

Comment:
I think some [Slack formatting](https://api.slack.com/docs/message-formatting#linking_to_urls) would add a nice touch here. Maybe include `draft.record.shelter` to include the text from the `Shelter` field like:

```ruby
send_slack_notification "New shelter added in <https://irma-api.herokuapp.com/drafts/#{draft.id}|#{draft.record.shelter}>"
```

You could also make it a call to action, ie "Admins, please review new shelter in  <https://irma-api.herokuapp.com/drafts/#{draft.id}|#{draft.record.shelter}>"